maths/gelu.py

```diff
@@ -0,0 +1,49 @@
+"""
```

Comment:
Please change the filename to `maths/gaussian_error_linear_unit.py` so that visitors do not need to guess.